go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/la5nta/wl2k-go
 
-go 1.11
+go 1.14
```

Comment:
Please revert this  - ref https://github.com/la5nta/pat/pull/222#pullrequestreview-525832837.

Reply:
Done! I should've remembered to do that here, too. Thanks